common/src/main/java/com/frostwire/bittorrent/BTDownload.java

```diff
@@ -241,7 +241,7 @@ public int getProgress() {
 
     @Override
     public boolean isComplete() {
-        return getProgress() == 100;
+        return getProgress() == 100 && !isSeeding();
```

Comment:
I think this might lead to a lot of bugs both in android and desktop.
Many places might be considering seeding transfers as complete.